### PR TITLE
add blossom-ci

### LINKED ISSUE #417: CI/CD: Move Weekly Build GH Action to Azure Pipeline
This was not required in the 0.2.0 release but still needs to be completed to transition all CI to Azure.

### PR DESCRIPTION
**Pull request by YanxuanLiu**
Fix #417 

enable blossom-ci for this repo, then move pre-merge tests to blossom

### CODE REVIEW

**Comment by tangy5** [comment]
Thanks, @YanxuanLiu , let me know if there are any question.

File: .github/workflows/blossom-ci.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# A workflow to trigger ci on hybrid infra (github + self hosted runner)
```
Comment (pxLi) [inline .github/workflows/blossom-ci.yml:1]: copyright header plz

**Review by pxLi — APPROVED** [review]
+1, please help verify and adjust if any problem during testing, thanks!

**Comment by tangy5** [comment]
> +1, please help verify and adjust if any problem during testing, thanks!

Thanks, I will verified related GPU tests.

**Review by tangy5 — APPROVED** [review]

**Pull request merged**